### PR TITLE
ci(admission-controller,agent,kspm-collector,node-analyzer,rapid-response,sysdig-deploy): add trigger to helm chart validation tests upon PR

### PR DESCRIPTION
**Pull request by macuna-sd**
## What this PR does / why we need it:

This PR creates a new github actions pipeline that will be triggered when changes are done to charts. This pipeline in turn triggers helm-chart-validation pipeline.

## Checklist

- [X] Title of the PR starts with type and scope, (e.g. feat(agent,node-analyzer,sysdig-deploy):)

### CODE REVIEW

File: .github/workflows/chart-automation-tests.yaml [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+on:
+  pull_request:
+    branches:
+      - master
+    paths:
+      - 'charts/sysdig-deploy/**'
+      - 'charts/admission-controller/**'
+      - 'charts/node-analyzer/**'
+      - 'charts/kspm-collector/**'
+      - 'charts/rapid-response/**'
```
Comment (aroberts87) [inline .github/workflows/chart-automation-tests.yaml:12]: You're missing the agent chart. 🙂

File: .github/workflows/chart-automation-tests.yaml [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+
+jobs:
+  helm-charts-validation-test:
+    runs-on: tools-runner
```
Comment (aroberts87) [inline .github/workflows/chart-automation-tests.yaml]: Is this a custom runner that needs to be created before this is merged?

File: .github/workflows/chart-automation-tests.yaml (hunk 1 above)
Comment (macuna-sd) [reply .github/workflows/chart-automation-tests.yaml:12]: sorry what do you mean by that?

File: .github/workflows/chart-automation-tests.yaml (hunk 2 above)
Comment (macuna-sd) [reply .github/workflows/chart-automation-tests.yaml]: you may be right...

File: .github/workflows/chart-automation-tests.yaml (hunk 1 above)
Comment (aroberts87) [reply .github/workflows/chart-automation-tests.yaml:12]: You need to add
```
- 'charts/agent/**'
```
🙂

File: .github/workflows/chart-automation-tests.yaml (hunk 2 above)
Comment (aroberts87) [reply .github/workflows/chart-automation-tests.yaml]: Is there anything special on that runner? If there isn't anything there that is proprietary you could just use something like `ubuntu-latest` and use apt to install anything that you need during each run. 🤷🏻‍♂️

**Pull request closed without merging**